[new release] opam-check-npm-deps (4.0.1)

jchavarri opened the pull request:
An opam plugin to check for npm depexts inside the node_modules folder

- Project page: <a href="https://github.com/ahrefs/opam-check-npm-deps">https://github.com/ahrefs/opam-check-npm-deps</a>

##### CHANGES:

- Fix syntax error after reason/ppx_let upstream changes.

jmid reviewed (commented):
CI for `opam-check-npm-deps.4.0.1` failed `to get sources of opam-check-npm-deps.4.0.1: HTTP error code 404` on some runs - but not on others. Something like this could happen, e.g., if there were changes to the source URL underway.
To ensure that everything is in order now, it would probably be good to do a dummy push... :pray: 

Also: would you consider adding an `x-maintenance-intent` entry?
https://github.com/ocaml/opam-repository/blob/master/governance/policies/archiving.md

jchavarri commented:
> CI for `opam-check-npm-deps.4.0.1` failed `to get sources of opam-check-npm-deps.4.0.1: HTTP error code 404` on some runs - but not on others. Something like this could happen, e.g., if there were changes to the source URL underway.

Sorry, I messed up the release, and fixed it thinking the build had not started fetching the sources yet, but it seems it did. Is it possible to re-run it again?

> Also: would you consider adding an `x-maintenance-intent` entry?
> https://github.com/ocaml/opam-repository/blob/master/governance/policies/archiving.md

I can for sure. Added in last commit.
